lib/file-helper.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const fs = require('fs')
+const nunjucks = require('nunjucks')
+
+// This helper function takes a path of a file and
+// returns the contents as string
+exports.getFileContents = path => {
+  let fileContents
+  try {
+    fileContents = fs.readFileSync(path)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      console.log(err.message)
+    } else {
+      throw err
+    }
+  }
+  return fileContents.toString()
+}
+
+// This helper function takes a path of a nunjucks file and
+// returns the first macro code block inside that file
+exports.getRawMacro = path => {
+  let fileContents = this.getFileContents(path)
+
+  let macro
+  try {
+    macro = fileContents.split('{{')[1]
```

Comment:
Presumably this is so we don't get the frontmatter or the include? Could definitely do with a comment to explain what this is trying to achieve.

Reply:
Yes, both the markdown data and the include(s).
I'll add a comment.

Reply:
This won't work if the example starts with HTML 😞 
We need a better delimiter (this was another reason for using the `{macro}`).

Reply:
Aah of course. Can we get rid of the frontmatter and leave the include in for now? We can find a way to remove the include later?